docs/release-notes/.Language/preview.md

```diff
@@ -4,4 +4,10 @@
 
 ### Fixed
 
+* Allow extension methods without type attribute work for types from imported assemblies. ([PR #16368](https://github.com/dotnet/fsharp/pull/16368))
+* Enforce AttributeTargets on let values and functions. ([PR #16692](https://github.com/dotnet/fsharp/pull/16692))
```

Comment:
These shouldn't be here, but in 9 changelog

Reply:
cool.

Reply:
done